### PR TITLE
876723-UG content for GetDataPoints support in .NET MAUI Cartesian chart

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description**

Prepared UG content for GetDataPoints support in .NET MAUI Cartesian chart

### CODE REVIEW

File: MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Get the data point collection based on region
+
+[CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html) provides the following methods to get a collection of data under a particular region.
+
+* [`GetDataPoints(Rect rectangle)`]() - Method to get the collection of data that fall inside the given rectangle region.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md:14]: Gets the collection of data that falls inside the given rectangle region.

File: MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+control: SfCartesianChart
+documentation: ug
+---
+
+# Get the data point collection based on region
+
+[CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html) provides the following methods to get a collection of data under a particular region.
+
+* [`GetDataPoints(Rect rectangle)`]() - Method to get the collection of data that fall inside the given rectangle region. 
+* [`GetDataPoints(double startX, double endX, double startY, double endY)`]() - Method to get the data points from the given range.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md:15]: Gets the collection of data from the given axis visible range.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]
Address the comments.

File: MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md]: Modified the content

File: MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md]: Modified the content

**Comment by EzhilarasanElangovan31** [comment]
> Address the comments.

@Saravanan-Madhesh sir, Addressed the comments.

**Review by Saravanan-Madhesh — APPROVED** [review]
Fine

**Pull request merged**